zmon-agent.py

```diff
@@ -279,54 +279,38 @@ def main():
                 if not e["id"] in current_entities:
                     to_remove.append(e["id"])
 
+            if os.getenv('zmon_user', None) is not None:
```

Comment:
I dislike the repeated "None", also it should consider empty strings as "False".

Simply use:

```python
if os.getenv('zmon_user'):
   ...